src/gateway/upgrade.md

```diff
@@ -397,10 +395,11 @@ to as the
 
 {:.important}
 > **Important**:
+TODO confirm this, and if false, remove this line. If true we need to change all the below documentation to reference EE-only instructions...
```

Comment:
🚧 🚧 🚧 
I want to get the answer to this before merging this PR

Reply:
I think this is mostly a product/political decision.  Blue/green upgrades on CE would require a compatibility 2.8.x release that enables 2.8.x DP nodes to work after the `up` migration part has run.  We're going to make such a release for EE, but I don't think that the same is planned for CE.

Reply:
Talked with Rob - I think there was miscommunication. CE does support no-downtime (blue/green) upgrades, but does not support 3.0 CP to 2.8 DP compatibility. Users could shut down their 2.8 CP but leave the 2.8 DPs serving traffic while bringing up 3.0 CP and DP nodes, then migrate traffic to the 3.0 cluster to do a no downtime deploy.